application/dash_application/assets/styles.css

```diff
@@ -377,6 +373,13 @@ input[type="button"] {
     box-sizing: border-box;
 }
 
+.sidebar .miso-qc-button {
+    float: left;
+    width: 30%;
+    margin-left: 2.5%;
+    margin-right: 2.5%;
```

Comment:
I know 3(30) + 2(2.5) != 100, but when I set these to 5, they'd overflow, and i couldn't identify the margin or padding that was ruining it

Reply:
CSS is supposed to make sense? If it bothers you, maybe `calc`. I'm good with it.